src/garage/envs/bullet/bullet_env.py

```diff
@@ -1,16 +1,126 @@
+            is_image (bool): True if observations contain pixel values,
+                false otherwise. Setting this to true converts a gym.Spaces.Box
+                obs space to an akro.Image and normalizes pixel values.
+
+        """
+        if not env:
+            # 'RacecarZedBulletEnv-v0' environment enables rendering by
+            # default, while pybullet allows only one GUI connection at a time.
+            # Setting renders to False avoids potential error when multiple
+            # of these envs are tested at the same time.
+            if env_name == 'RacecarZedBulletEnv-v0':
+                env = gym.make(env_name, renders=False)
+            else:
+                env = gym.make(env_name)
+
+        # Needed for deserialization
+        self._env = env
+        self._env_name = env_name
+
+        super().__init__(env)
+        self.action_space = akro.from_gym(self.env.action_space)
+        self.observation_space = akro.from_gym(self.env.observation_space,
+                                               is_image=is_image)
+        self.__spec = EnvSpec(action_space=self.action_space,
+                              observation_space=self.observation_space)
+
+    @property
+    def spec(self):
+        """Return the environment specification.
+
+        This property needs to exist, since it's defined as a property in
+        gym.Wrapper in a way that makes it difficult to overwrite.
+
+        Returns:
+            garage.envs.env_spec.EnvSpec: The envionrment specification.
+
+        """
+        return self.__spec
+
+    def close(self):
+        """Close the wrapped env."""
+        #  RacecarZedBulletEnv-v0 environment doesn't disconnect from bullet
+        #  server in its close() method.
+        #  Note that disconnect() disconnects the environment from the physics
+        #  server, whereas the GUI window will not be destroyed.
+        #  The expected behavior
+        if self.env.env.spec.id == 'RacecarZedBulletEnv-v0':
+            # pylint: disable=protected-access
+            if self.env.env._p.isConnected():
+                self.env.env._p.disconnect()
+        self.env.close()
+
+    def reset(self, **kwargs):
+        """Call reset on wrapped env.
+
+        This method is necessary to suppress a deprecated warning
+        thrown by gym.Wrapper.
+
+        Args:
+            kwargs: Keyword args
+
+        Returns:
+            object: The initial observation.
+
+        """
+        return self.env.reset(**kwargs)
+
+    def step(self, action):
+        """Call step on wrapped env.
+
+        This method is necessary to suppress a deprecated warning
+        thrown by gym.Wrapper.
+
+        Args:
+            action (object): An action provided by the agent.
+
+        Returns:
+            object: Agent's observation of the current environment
+            float : Amount of reward returned after previous action
+            bool : Whether the episode has ended, in which case further step()
```

Comment:
i think pre-commit will catch this, but the form is `bool: Whether...` (i.e. no spaces after the colon)

formatting is important so that the API doc parser can work.